dashboard/test/ui/features/applab/dataBlocks.feature

```diff
@@ -0,0 +1,13 @@
+@no_ie
+@dashboard_db_access
+Feature: App Lab Data Blocks
+
+  Background:
+    Given I sign in as a student
+    And I am on "http://studio.code.org/s/allthethings/stage/18/puzzle/8?noautoplay=true"
```

Comment:
It's not clear from looking at this file exactly what it is that we're testing. I believe that's because the bulk of the logic is in the level itself? Might be worth a short comment explaining what/how this feature tests.